03week/airplane/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+<!--Sky Mendes, Laura Lamb & Elvira Joubert -->
+<head> 
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+<meta charset="utf-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>Airline Page</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" type="text/css" media="screen" href="./css/style.css" />
+
+</head>
+<header>
+<section class="top">
+<div class="logo"><img src= "img/logo.png" alt= "logo">
+</div> 
+
+<div class= "navbar">
+<nav>
+<a class="home" href="#"><img src="./img/home-icon.png" alt= "home"> Home</a>
+
+<a class="home" href="#"><img src="./img/deal-icon.png" alt="travel deal"> Travel Deals </a>
+
+<a class="home" href="#"><img src="./img/schedule-icon.png" alt="flight schedule"> Flight Schedule</a>
+
+<a class="home" href="#"><img src="./img/login-icon.png" alt="register login"> Register/Login</a>
+</nav>
+</div>
+</section>
+</header>
+
+<section> <!-- This is the box content-->
+<div class="divbox">
+<div class="iconbar"> <!-- img icon bar-->
+<button class="button" id="flight"></button>
+<button class="button" id="hotel"></button>
+<button class="button" id="rentalcar"></button>
+</div>
+</div>
+<body>
+<div class="box">
+<div class="text-box">
+<input type="text" class="depart-field" placeholder="Depart" required /> 
+<input type="text" class="arrive-field" placeholder="Arrive" requires /> 
+<div class="clearfix"></div>
+<input type="date" class="date" placeholder="mm/dd/yyyy" required />
+<input type="date" class="date" placeholder="mm/dd/yyyy" requires />
+</div>
+<div class="clearfix"></div>
+<p class="passengers">Passengers</p> 
+<select>
+<optgroup label="numbers" class="numbers">
+<option value="1">1</option>
+<option value="2">2</option>
+<option value="3">3</option>
+<option value="4">4</option>
+<option value="5">5</option>
+</optgroup>
+</select>
+
+<div class="radio">
+<input type="radio" name="flight" value="return" checked /><label for "return">Return</label>
+<input type="radio" name="flight" value="oneway" /><label for "oneway">One Way</label>
+</div>
+<div class="clearfix"></div>
+<input type="submit" value="Search Flights">
+</div> <!---end of box class-->
+</section>
+
+
+</body>
+</html>
```

Comment:
I want some indentation for the whole file please. I think there is an extension that can help you do that.